pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -576,8 +618,10 @@ func (gce *GCECloud) EnsureLoadBalancer(name, region string, requestedIP net.IP,
 		}
 		// End critical section.  It is safe to release the static IP (which
 		// just demotes it to ephemeral) now that it is attached.  In the case
-		// of a user-requested IP, the 'defer' won't be installed anyway.
-		releaseStaticIP = true
+		// of a user-requested IP, the "is user-owned" flag will be set,
+		// preventing it from actually being released.
+		isSafeToReleaseIP = true
```

Comment:
Nevermind -- looking at each of the cases it's used in, this is right. `isUserOwnedIP` is really managing that second bit, so disregard this